src/compute_pressure.cpp

```diff
@@ -132,6 +133,11 @@ ComputePressure::ComputePressure(LAMMPS *lmp, int narg, char **arg) :
   vector = new double[size_vector];
   nvirial = 0;
   vptr = nullptr;
+
+  // KOKKOS
```

Comment:
Is this what affected the Kokkos overhead slowdown?

Reply:
Yes, `compute pressure` doesn't use any atom arrays, and needed to be marked as such, or else a ton of data movement between GPU and CPU occurred due to this change: [https://github.com/lammps/lammps/commit/8827dec5a91472eab3063fdb3108ab03db0dd327#diff-3b9190ecdb0f077e35caa96545a[…]8d6d765f335e39e30494911b8d6fR204](https://github.com/lammps/lammps/commit/8827dec5a91472eab3063fdb3108ab03db0dd327#diff-3b9190ecdb0f077e35caa96545a2db01ff558d6d765f335e39e30494911b8d6fR204)